tap_shopify/streams/locations.py

```diff
@@ -0,0 +1,37 @@
+import shopify
+from singer import utils
+from tap_shopify.streams.base import (Stream, shopify_error_handling)
+from tap_shopify.context import Context
+
+class Locations(Stream):
+    name = 'locations'
+    replication_object = shopify.Location
+
+    @shopify_error_handling
+    def get_locations_data(self):
+        location_page = self.replication_object.find()
```

Comment:
Are there any parameters being missed for find like API for inventory levels?

Reply:
No, the location API [documentation](https://shopify.dev/api/admin-rest/2021-07/resources/location) does not have the parameters like the inventory levels.